tests/xharness/Targets/MacTarget.cs

```diff
@@ -56,36 +53,16 @@ protected override void CalculateName ()
 				}
 			}
 		}
-			
-		protected override string ProjectTypeGuids {
-			get {
-				return "{A3F8F2AB-B479-4A4A-A458-A89E7DC349F1};" + LanguageGuid;
-			}
-		}
 
-		protected override string BindingsProjectTypeGuids {
-			get {
-				return "{810C163F-4746-4721-8B8E-88A3673A62EA}";
-			}
-		}
+		protected override string ProjectTypeGuids => "{A3F8F2AB-B479-4A4A-A458-A89E7DC349F1};" + LanguageGuid;
```

Comment:
These type of changes make the PR absolutely horrible to review, because I spend an extraordinary amount of time wondering what changed, just to figure out that nothing actually did. Please do not refactor code just to refactor code.

Reply:
We are refactoring the code as part of this effort though.

There are no functional changes, just a bit heavier IDE auto-formatting, so shouldn't be that hard to review... but if you really feel strong about this, I can revert the contents. Just that it's never going to be refactored after this and it makes the code way shorter to read in the future so will save time later (for a small price now)

Reply:
> There are no functional changes

The 32 failed tests show that something functional changed.

> but if you really feel strong about this, I can revert the contents

Yes, please.

> Just that it's never going to be refactored after this

If you want to refactor code, do it in a separate PR, and make it as clear as possible (PR description / commit message) that there should only be no functional changes.

It's always much easier to review two (or more!) small PRs than a single big one.

Reply:
Reverted.

> It's always much easier to review two (or more!) small PRs than a single big one.

It is easy but not across time zones when on a tight schedule and having changes heavily depend on each other because files are moved around a lot. But I agree it's better to have a single purpose PRs

Reply:
@premun I have made the same mistake, I will try to avoid fast fingers ;)